controllers/suite_util_test.go

```diff
@@ -599,3 +599,45 @@ func deleteBuildPipelineConfigMap(configMapKey types.NamespacedName) {
 		Fail(err.Error())
 	}
 }
+
+func waitServiceAccount(serviceAccountKey types.NamespacedName) {
```

Comment:
Could we improve this function name? Maybe something like `waitForServiceAccount()`?

Reply:
I followed pattern used, see other functions with `wait` prefix, like `waitSecretGone`, etc.